assets/terraform-modules/worker/variables.tf

```diff
@@ -4,6 +4,10 @@ variable "ca_cert" {
   description = "Kubernetes CA certificate needed in the kubeconfig file."
 }
 
+variable "cluster_name" {
```

Comment:
>worker module: add `cluster_name` variable

This on it's own brings no value. It should be added together with code where it's used.